site/docs/v1/tech/samlv2/_saml_limitations.adoc

```diff
@@ -1,15 +1,24 @@
-FusionAuth supports SAML both as an Identity Provider (IdP) and as a Service Provider (SP). 
+FusionAuth supports SAML both as an Identity Provider (IdP) and as a Service Provider (SP). The IdP implementation has certain limitations. 
 
-The IdP implementation has certain limitations. Version `2.0` is supported; other SAML versions are unsupported.
+Version `2.0` is supported; other SAML versions are unsupported.
 
-FusionAuth supports two `NameIDPolicy` values:
+FusionAuth supports the following `NameIDPolicy` values:
 
 * `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress`
+* `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`
 * `urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified`
 
-Please examine the `NameIDPolicy` of the software package for which you are configuring FusionAuth as the IdP to determine if these `NameIDPolicy` values are supported. 
+To determine the correct value for `NameIDPolicy`, please examine the supported values of the software package for which you are configuring FusionAuth as the IdP.
 
-There is an https://github.com/FusionAuth/fusionauth-issues/issues/522[open feature request] to support `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent` but this `NameIDPolicy` is currently unsupported.
+FusionAuth won't fail validation on any `NameIDPolicy` value. If the requested `NameIDPolicy` is `persistent`, FusionAuth will use that and return the User's unique id (`user.id`). In all other cases, the User's email address (`user.email`) will be used for the name Id.
```

Comment:
I looked at the code again, I lied to you. We will fail validation if it is not one of these three options. Using `unspecified` or `email` will result in using `email` for backwards compatibility.